torchvision/transforms/functional.py

```diff
@@ -1484,3 +1484,65 @@ def equalize(img: Tensor) -> Tensor:
         return F_pil.equalize(img)
 
     return F_t.equalize(img)
+
+
+def elastic_transform(
```

Comment:
Should this be added to the `__init__.py` or not?

Reply:
No, it is ok like that :)